tests/_data/project.yaml

```diff
@@ -84,7 +84,7 @@ siteGroups:
     name: 'Craft CMS Test Site Group'
 sites:
   2fa19d28-c359-4955-8b0d-35a35e3f1128:
-    baseUrl: $DEFAULT_SITE_URL
```

Comment:
@timkelty is there a reason we’re still making this change for tests?

Reply:
eh, just because it is the sole remaining reference to `$DEFAULT_SITE_URL` in the codebase. Seems like it should be either `@web`, `$PRIMARY_SITE_URL`, or `CRAFT_WEB_URL`.